xml/System/Span`1.xml

```diff
@@ -597,10 +668,10 @@
         <Parameter Name="segment" Type="System.ArraySegment&lt;T&gt;" Index="0" FrameworkAlternate="netcore-2.1" />
       </Parameters>
       <Docs>
-        <param name="arraySegment">To be added.</param>
-        <param name="segment">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="arraySegment">The array segment to be converted to a <see cref="T:System.Span`1" />.</param>
+        <param name="segment">The array segment to be converted to a <see cref="T:System.Span`1" />.</param>
+        <summary>Defines an implicit conversion of an <see cref="T:System.ArraySegment" /> to a <see cref="T:System.Span`1" />.</summary>
```

Comment:
The xml tag around ArraySegment needs an adjustment. It isn't showing up as a link.